cwm_worker_operator/domains_config.py

```diff
@@ -144,7 +144,7 @@ def __init__(self, data, domains_config, request_hostname=None, is_data_from_cac
         self.hostname_challenges = {}
         minio_extra_configs = data.get('minio_extra_configs', {})
         self.protocols_enabled = set()
-        for protocol in minio_extra_configs.pop('protocols-enabled', ['http', 'https']):
+        for protocol in minio_extra_configs.pop('protocols-enabled', ['http', 'https', 'disable_http', 'disable_https']):
```

Comment:
there is no change to the protocols-enabled field, it still contains just http / https (the enabled protocols)